[Home](index.md) | [Application & Fees](application.md) | [Timetable](timetable.md) | [Location, Accommodation, & Dining](location.md)

## What is Sing-a-Pore Camp 2018
Sing-a-Pore Camp 2018 is the seventh edition of the PoreCamp series, founded by Nick Loman, Matt Loose and Mick Watson in 2015. It is a five-day training bootcamp based around the Oxford Nanopore Technology (ONT)’s MinION sequencing system. The workshop will be a unique opportunity to get hands-on experience in generating and analysing MinION data from some of the leading experts in the field – Nick Loman, Josh Quick, Matt Loose and John Tyson will join local hosts Niranjan Nagarajan, TBC1 and TBC2 for five days of sequencing, bioinformatics, and general merriment.

## When and where will Sing-a-Pore Camp 2018 take place?
The workshop will run all day Tuesday, 17th September through all day Saturday, 22nd September, 2018. A detailed timetable is available [here] (timetable.md). We’ll be based at the Genome Institute of Singapore at the Genome Building, Biopolis, Singapore. For more details on getting to the Genome Building, click [here] (location.md).

## What is nanopore sequencing?
The [Oxford Nanopore MinION](https://nanoporetech.com/products/minion) is a USB-size DNA sequencer which is capable of producing 10-20 Gbase of sequence data with read lengths as high as hundreds of kilobases. One of its key advantages is to produce data in near-real time - the system has been used to monitor the [Ebola outbreak](http://www.nature.com/nature/journal/v530/n7589/full/nature16996.html) in 2015, the [Zika outbreak](http://zibraproject.github.io/) in 2016, and in July 2016 the MinION became the [first sequencer in space](http://www.nasa.gov/mission_pages/station/research/experiments/2181.html). The Porecamp instructors recently [sequenced a whole human genome](http://github.com/nanopore-wgs-consortium/NA12878) with this technology.

## Who should attend?
Anyone wanting hands-on training in library preparation, running the MinION instrument, and data analysis are welcome to attend. Participants do NOT require both wet-lab and bioinformatics skills to benefit from the course - training will be provided in both components. The course is open to anyone with an interest in the technology, whether they are students, researchers, or technologists. More details, as well as workshop costs, are available on the [Application and Fees page] (application.md).

## Course structure and programme
PoreCamp is currently run as a five-day course - see the timetable here. It is entirely hands-on. The aim is to provide hands-on training by researchers, for researchers in library preparation, running the instrument and the analysis of data, as well as the opportunity to meet and discuss applications with nanopore sequencing experts. As of Porecamp 2016, each course features a sequencing field trip to explore the portable capabilities of the device.

The programme and teaching materials for previous courses can be found at:
  * [PoreCamp2015](https://porecamp.github.io/2015/index.html) (14 - 18 December 2015, University of Birmingham)
  * [PoreCamp2016](https://porecamp.github.io/2016/index.html) (15 - 19 August 2016, University of Exeter, Penryn Campus, Cornwall)
  * [PoreCamp Australia](https://porecamp-au.github.io/) (8 - 10 February 2017, University of Melbourne)
  * [PoreCamp USA](http://www.txgen.tamu.edu/porecamp_usa/) (5 - 9 June 2017, Texas A&M University)
  * [PoreCamp2017](https://porecamp.github.io/2017/index.html) (3 - 7 July 2017, University of Birmingham)
  * [PoreCamp Vancouver](https://porecamp.github.io/vancouver/index.html) (7 - 11 May 2018, University of British Columbia)

During Sing-a-Pore Camp 2018, we will be sequencing multiple samples. Some of these will be provided by participants, while others will be prepared by the instructor team. Please do not count on Sing-a-Pore Camp 2018 as a source of data for your experiment and please do not sequence anything that is incredibly precious - if one of your samples is sequenced, it will be multiplexed with others on a run and you will not have a full flow cell’s worth of data. We cannot guarantee how much sequence data your sample will produce, and the sample will be used in its entirety - there will be none left. To streamline sequencing, please bring a sample with at least 1ug of extracted genomic DNA. We will not be sequencing metagenomic samples, nor human samples, during the workshop.

## Contact Us
**Please direct all questions about the workshop to [singaporecamp2018@gmail.com](mailto:singaporecamp2018@gmail.com)**

